addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/DialogAddField.java

```diff
@@ -163,7 +164,7 @@ protected void showNameRepeatedDialog() {
     protected void performAction() {
         field =
                 new FormHandlerParamField(
-                        getNameTextField().getText().toLowerCase(),
+                        getNameTextField().getText().toLowerCase(Locale.ROOT),
```

Comment:
The `FormHandlerParamField` already lowers case the name, this lower case should just be removed. There are other similar occurrences where the name is being lower cased when it already was.

Reply:
I think I've addressed all these. I also tweaked `FormHandlerParamField.setName(String)` to lower case the name when setting it. I don't see this used in this add-on but I figured that was safe measure to ensure calling code doesn't have to consider it.

Reply:
Actually, instead of lower casing so often what do you think about just changing the few comparisons to equalsIgnoreCase. The ValueGenerator already seems to do that, but the add dialog doesn't for some reason. But that seems like an easy change to me.

I guess we could leave it in the Param class because that would mean whatever is svaed/set is consistent and less work for equalsIgnoreCase. But even that might be me over thinking a bit.

Reply:
But that would allow duplicates in the UI (at least until is saved), right?

Reply:
Oh okay, hadn’t thought of it that way